server/rest/src/test/java/org/infinispan/rest/resources/AbstractRestResourceTest.java

```diff
@@ -59,19 +63,31 @@
    private final List<RestServerHelper> restServers = new ArrayList<>(NUM_SERVERS);
 
    protected boolean security;
+   protected boolean http2;
+   protected boolean ssl;
 
    protected CacheIgnoreManager ignoreManager;
 
    @Override
    protected String parameters() {
-      return "[security=" + security + "]";
+      return "[security=" + security + ", http2=" + http2 + ", ssl=" + ssl + "]";
    }
 
    protected AbstractRestResourceTest withSecurity(boolean security) {
       this.security = security;
       return this;
    }
 
+   protected AbstractRestResourceTest http2(boolean http2) {
```

Comment:
nitpick: instead of a `boolean`, I probably use `protocol(org.infinispan.client.rest.configuration.Protocol)` to be future proof :+1: